cliquet/resource.py

```diff
@@ -239,7 +238,8 @@ class BaseResource(object):
     mapping = ResourceSchema()
     """Schema to validate records."""
 
-    def __init__(self, request):
+    def __init__(self, request, context=None):
```

Comment:
Yep, tests fail without this. I could update them but I kinda like the fact that the context is optional.